package.json

```diff
@@ -29,35 +33,50 @@
     "electron:serve": "vue-cli-service electron:serve",
     "postinstall": "electron-builder install-app-deps",
     "postuninstall": "electron-builder install-app-deps",
+    "electron:generate-icons": "electron-icon-builder --input=./public/icon.png --output=build --flatten",
     "ui": "vue ui"
   },
   "main": "background.js",
   "dependencies": {
-    "@node-wot/binding-coap": "^0.7.1",
```

Comment:
actually removing the hat here is important. Also testing with 0.7.4 would be good since it has bugfixes